source/tutorials/utilities.rst

```diff
@@ -177,6 +177,78 @@ run, using ``inspect-metadata``, we can learn all about it:
 
    How many IDs are there? How many columns? Are there any categorical columns? Why?
 
+Casting Metadata
+................
+
+In the :doc:`Metadata tutorial <metadata>` we learned about column types and utilizing the 
+``qiime tools cast-metadata`` tool to specifiy column types within a provided metadata file.
+Below we will go through a few scenarios of how this tool can be used, and some common mistakes
+that may come up.
+
+We'll start by first downloading some sample metadata. **Note**: This is the same sample
+metadata used in the **Inspect Metadata** section, so you can skip this step if you have
+already downloaded the ``sample_metadata.tsv`` file from above.
+
+.. download::
+   :url: https://data.qiime2.org/2021.8/tutorials/pd-mice/sample_metadata.tsv
+   :saveas: sample_metadata.tsv
+
+In this example, we will cast the ``days_post_transplant`` column from ``numeric`` to ``categorical``,
+and the ``mouse_id`` column from ``categorical`` to ``numeric``. The rest of the columns contained
+within our metadata will be left as-is.
+
+.. command-block::
+   :stdout:
+
+   qiime tools cast-metadata sample_metadata.tsv \
+   --cast days_post_transplant:categorical --cast mouse_id:numeric
+
+If the ``--output-file`` flag is enabled, the specified output file will contain the modified column types that we cast above
+for the original ``sample_metadata.tsv``.
+
+If you do not wish to save your casted metadata to an output file, you can omit the ``--output-file`` parameter and
+the results will be output to ``sdtout`` (as shown in the example above).
+
+The ``--ignore-extra`` and ``--error-on-missing`` flags are used to handle casted columns not contained within the original
+metadata file, and columns contained within the metadata file that aren't included in the cast call, respectively.
+We can take a look at how these flags can be used below:
+
+In the first example, we'll take a look at utilizing the ``--ignore-extra`` flag when a column is cast that is not included
+within the original metadata file. Let's start by looking at what will happen if an extra column is included and this flag
+is not enabled.
+
+.. command-block::
+   :stderr:
+   :allow-error:
+
+   qiime tools cast-metadata sample_metadata.tsv \
+   --cast spleen:numeric
+
+Notice that the ``spleen`` column included in the cast call results in a raised error. If we want to ignore any extra columns that
+are not present in the original metadata file, we can enable the ``--ignore-extra`` flag.
+
+.. command-block::
+   :stdout:
```

Comment:
On second thought, maybe we should drop the `:stdout:` here and just let folks know that we are omitting the output --- it is long and doesn't really serve much purpose being printed here a second time.